pkg/vm/engine/tae/containers/tools.go

```diff
@@ -29,12 +29,13 @@ func GetValueFrom[T types.FixedSizeT](tool *CodecTool, i int) (v T) {
 }
 
 type CodecTool struct {
```

Comment:
I think you can remove this class. It is useless now.

Reply:
sure.